bluesky/run_engine.py

```diff
@@ -1733,9 +1733,12 @@ def fut_fac() -> awaitable:
                'This must work multiple times'
 
         """
+
         futs, = msg.args
         futs = [asyncio.ensure_future(f()) for f in futs]
-        await asyncio.wait(futs, **self._loop_for_kwargs, **msg.kwargs)
+        futs = await asyncio.wait(futs, **self._loop_for_kwargs, **msg.kwargs)
+        if futs[1]:
```

Comment:
Nit: I'd probably unpack to make it a little clearer what `futs[1]` is

```suggestion
        _, pending = await asyncio.wait(futs, **self._loop_for_kwargs, **msg.kwargs)
        if pending:
```